### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by forceeeee** [Gelbpunkt/IdleRPG#271]
**Is your feature request related to a problem? Please describe.**
The majority of questions asked in #support are already answerd in wiki, but no one uses it.

**Describe the solution you'd like**
There are two reasons as to why people don't use our wiki.  1. [People don't know we have a wiki]
2.[ People just don't want to use wiki]

**Describe alternatives you've considered**
Make a rule that you have to use wiki before asking a question in support, if this isn't a rule, then who will use wiki? That's the reason the first wiki died, and this one will die too. So unless there is a way to make people use wiki, it will be just an unused project. Even baumi had to make an event so he can make wiki active again and so people use wiki, we can't just bribe people into using wiki by using crates and money.

**Additional context**
I hope you take this into consideration.

**Comment by maxx0911** [comment]
I think a common factor is also that people don't have a way of accessing the wiki from the bot itself. That's why I proposed [this file change](https://github.com/Gelbpunkt/IdleRPG/pull/264), which is merged but not yet updated in the bot.

Otherwise I would suggest not being too strict about it, eventually, if the wiki covers **everything**, people like me in the support team wouldn't have anything to do most of the time ;p

**Comment by numbermaniac** [comment]
There is a `$wiki` command, but the output tends to be a bit difficult to read because it lacks all formatting.

**Comment by forceeeee** [comment]
It would be nice if you can make a $wiki that just outputs answers from the Idle wiki. And there is a page you can flip through to choose what you want to read about.

**Comment by maxx0911** [comment]
You are literally thinking of the `$idlewiki` command.
[This command](https://github.com/Gelbpunkt/IdleRPG/blob/v4/cogs/miscellaneous/__init__.py#L726) takes input like `$wiki Profile` and outputs the text from the idlewiki page.
Maybe I understood it wrong, but this is what you mean, right?

**Comment by forceeeee** [comment]
Yes, and I also said it would be nice to have a flipbook type of thing and you look through all the different types of wiki posts. And like number said, the formatting is bad. It's a mere copy paste of what it says. Would be nice if it can have a rework

**Comment by Gelbpunkt** [comment]
it is a mere copy paste because it's what's actually in the wiki, filtered so it's human readable

**Comment by forceeeee** [comment]
Is it possible to make the formatting a **little** better?
